packages/adapter-cloudflare-workers/files/entry.js

```diff
@@ -25,18 +26,19 @@ async function handle(event) {
 		}
 	}
 
-	// fall back to an app route
-	const request = event.request;
-	const request_url = new URL(request.url);
+	const parsedURL = new URL(request.url);
```

Comment:
CFW `request` doesn't expose the usual props. That's why I did it differently for this adapter. [Doc](https://developers.cloudflare.com/workers/runtime-apis/request#properties)